### ISSUE TITLE
Group checkboxes lack contrast in dark theme

### ISSUE THREAD

**Issue by Korb** [CatimaLoyalty/Android#1971]
![Screenshot_2024_0704_203348](https://github.com/CatimaLoyalty/Android/assets/3509797/33dc200b-4518-484a-9d14-8e84fd0fa856)

![2024-07-04_20-40-32](https://github.com/CatimaLoyalty/Android/assets/3509797/421b6168-5add-4b8b-ae38-cc9460c34c61)

> Foreground: #000000
> Background: #282829
> Contrast ratio: 1.426:1
> (...)
> 1.4.11 Non-text Contrast (AA)
>     Fail for UI components and graphical objects

Catima 2.30.0 f-droid
